test/integration/generated/readmeFileChecker/src/keyVaultClientContext.ts

```diff
@@ -26,6 +26,15 @@ export class KeyVaultClientContext extends coreClient.ServiceClient {
       requestContentType: "application/json; charset=utf-8"
     };
 
+    const packageDetails = `@azure/keyvault-secrets/1.0.0-preview1`;
```

Comment:
This format doesn't seem to be consistent with existing Track2 packages. For example `@azure/search-documents` has as prefix `azsdk-js-search-documents/${SDK_VERSION}` ( [link](https://github.com/Azure/azure-sdk-for-js/blob/a760eadea01c1c11960ad9071f704340fd867ea5/sdk/search/search-documents/src/searchClient.ts#L119))

Other packages follow the same as search, we may need to adjust to that. Also, it would be a good idea to update the guidelines with the format we want for user-agent, I didn't find anything specific about that ([link](https://azure.github.io/azure-sdk/typescript_implementation.html#general-distributed-tracing))

Reply:
Here is the guideline on user agent strings: https://azure.github.io/azure-sdk/general_azurecore.html#telemetry-policy

We need to have `azsdk-js-` followed by the package name without the `@azure/`

Reply:
Thanks @joheredi. I will modify the user agent string to `azsdk-js-<package_name>` in the next commit